ModerationPolicy.md

```diff
@@ -0,0 +1,5 @@
+### Express WG Moderation Policy
+
+The [Node.js Moderation Policy] applies to this WG.
+
+[Node.js Moderation Policy]: https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md
```

Comment:
@Fishrock123 most of this is just currently boilerplate from the TSC base
docs. This would all be stuff the WG would need to decide on whether to
keep or change. I'm just providing the base documents to bootstrap things.
On Jan 29, 2016 1:32 PM, "Jeremiah Senkpiel" <notifications@github.com>
wrote:

> In ModerationPolicy.md
> <https://github.com/strongloop/express/pull/2871#discussion_r51318658>:
>
> > @@ -0,0 +1,5 @@
> > +### Express WG Moderation Policy
> > +
> > +The [Node.js Moderation Policy] applies to this WG.
> > +
> > +[Node.js Moderation Policy]: https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md
>
> I'm not sure why this is here? We don't do this on other repos, at least
> right now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strongloop/express/pull/2871/files#r51318658>.
>